bin/gobblin.sh

```diff
@@ -283,18 +291,18 @@ elif [[ -f ${GOBBLIN_CONF}/log4j.properties ]]; then
     LOG4J_OPTS="-Dlog4j.configuration=$LOG4J_FILE_PATH"
 fi
 
+#Create required directories
 if [[ ! -d "$GOBBLIN_LOGS" ]]; then
     mkdir -p $GOBBLIN_LOGS
 fi
 
-GC_OPTS=''
-if [[ ${ENABLE_GC_LOGS} -eq 1 ]]; then
-    GC_OPTS+="-XX:+UseConcMarkSweepGC -XX:+UseParNewGC -XX:+UseCompressedOops "
-    GC_OPTS+="-XX:+PrintGCDetails -XX:+PrintGCDateStamps -XX:+PrintTenuringDistribution "
-    GC_OPTS+="-XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=$GOBBLIN_LOGS/ "
-    GC_OPTS+="-Xloggc:$GOBBLIN_LOGS/gobblin-$GOBBLIN_MODE-gc.log "
+if [[ ! -d "$GOBBLIN_WORK_DIR" ]]; then
+    mkdir -p $GOBBLIN_WORK_DIR
 fi
 
+if [[ ! -d "$GOBBLIN_JOB_CONFIG_DIR" ]]; then
+    mkdir -p $GOBBLIN_JOB_CONFIG_DIR
+fi
 
```

Comment:
If these are initiated using `gobblin-env.sh`, and called in `gobblin.sh` and never overwritten before the directories are created, would it be possible to modify the env variables?

Reply:
do u mean if the change in env variables in `gobblin-env.sh` will get reflected and get dir created accordingly? yes. since `gobblin.sh` includes the `gobblin-env.sh`. I have added option to override it from user input as well, i think it will be useful to maintain any backward compatibility.